feat: adding the OOP platformer project to new JS curriculum (#50385)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-oop-by-building-a-platformer-game/64acedb5f59c0c8d43e96aa4.md

---
id: 64acedb5f59c0c8d43e96aa4
title: Step 40
challengeType: 0
dashedName: step-40
---

# --description--

> As the player starts and stops in the game, you will need to keep track of when the left and right arrow keys are pressed. 

Reviewer: ```suggestion
To manage the player's movement in the game, you will need to monitor when the left and right arrow keys are pressed.
```
